tools/github/write_current_source_version.py

```diff
@@ -8,41 +8,55 @@
 # Note: This script can only be executed from project root directory of an extracted "release"
 # tarball.
 
+import argparse
 import json
 import pathlib
 import sys
 import urllib.request
 
 if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Write current source version')
+    parser.add_argument(
+        '--skip_error_in_git',
+        dest='skip_error_in_git',
+        help='Skip returning error on exit when the current directory is a git repository.',
+        action='store_true')
+    args = parser.parse_args()
+
     # Simple check if a .git directory exists. When we are in a Git repo, we should rely on git.
-    if pathlib.Path(".git").exists():
+    if pathlib.Path('.git').exists():
```

Comment:
im wondering why change the quotation marks - we dont have a hard rule about this - but i think how you had it is more consistent

Reply:
Goctha. Sorry. Seems like my formatter did it :(.

Reply:
@phlax I have updated the quotation marks.